scripts/publish

```diff
@@ -9,7 +9,7 @@ fi
 function usage() {
     echo -n \
         "Usage: $(basename "$0")
-Publish all stactools.modiss.
+Publish stactools-modis.
```

Comment:
Does this change also relate to the template? I.e, did the script publish all stactools packages in the past (before the packages were broken out)?

Reply:
Yup, it did, we should update the template.